about_us.html

```diff
@@ -1,150 +1,131 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Senior Design Semester 1</title>
-    <link rel="stylesheet" target="_blank" href="style.css" />
-  </head>
-  <body class="body">
-    <header class="header">
-      <h1>Senior Design</h1>
-    </header>
 
-    <nav class="navbar">
-      <ul>
-        <li><a href="index.html">Home</a></li>
-        <li><a href="project_details.html">Project Details</a></li>
-        <li><a href="#about_us">About Us</a></li>
-      </ul>
-    </nav>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Senior Design Semester 1</title>
+  <link rel="stylesheet" target="_blank" href="style.css" />
+</head>
+
+<body class="body">
+  <header class="header">
+    <h1>SCORE: Student Code Online Review and Evaluation</h1>

```

Comment:
I'd Remove the SCORE:

Reply:
But that's part of our official name

Reply:
Its not though, the name is Student Code Online Review and Evaluation. That gets abbreviated to SCORE. 
SCORE: Lorem Ipsum, looks improper on the site

Reply:
I think we should keep SCORE